tests/local/local.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+from selenium.webdriver.common.by import By
+
+def test_local(selenium):
+  selenium.get('http://bs-local.com:45691/check')
+
+  if selenium.find_element(By.CSS_SELECTOR, 'body').text == 'Up and running':
+    selenium.execute_script('browserstack_executor: {"action": "setSessionStatus", "arguments": {"status":"passed","reason": ""}}')
```

Comment:
Let's add the `reason` fields to the session status update commands throughout the samples.

Reply:
fixed https://github.com/browserstack/pytest-browserstack/pull/2/commits/11987010959e762af5716115736e2f780e58562f